MyBiggerBuffs.lua

```diff
@@ -77,30 +81,57 @@ local function createBuffFrames(frame)
   end
 end
 
+function checkFrame(frame)
+    if not issecurevariable(frame, "action") and not InCombatLockdown() then
+        frame.action = nil
+        frame:SetAttribute("action");
+    end
+end
+
+
 local function activateMe()
   if started == true then
     return
   end
   started = true
+  for _, frame in ipairs(ActionBarButtonEventsFrame.frames) do
+	hooksecurefunc(frame, "UpdateAction", checkFrame);
+  end
+  hooksecurefunc('CompactUnitFrame_UpdateAll', function(frame)
+		if frame:IsForbidden() then
+			return
+		end
+		local name = frame:GetName()
+		if not name or not name:match('^Compact') then
+			return
+		end
+		if InCombatLockdown() then
+			if not UpdateTable[frame] then

```

Comment:
This throws a nil error - the table need to be declared. And you also need to actually do something with the table for it to be useful. It seems to be taken from BigDebuffs where they use it to call the createBuffFrames(frame) on PLAYER_REGEN_ENABLED (= when you get out of combat) which is missing here.

Reply:
Good call. I got rid of the table thing. I'm not sure where the original idea comes from. I saw this from an addon Sortgroup first.